docs/index.md

```diff
@@ -46,6 +46,7 @@ Besides running Zigbee2MQTT on [bare-metal Linux](getting_started/running_zigbee
 * [How to support new devices](how_tos/how_to_support_new_devices.md)
 * [How to support new Tuya compatible devices](how_tos/how_to_support_new_tuya_devices.md)
 * [How to switch to the dev branch](how_tos/how-to-switch-to-dev-branch.md)
+* [DIYRuZ catalog of DiY zigbee devices](https://diyruz.github.io/)

```

Comment:
Maybe fits better under integration section or change the name to "How to create your own Zigbee devices"?

Reply:
done